### PR TITLE
Update extension gallery files

### PR DESCRIPTION
**Pull request by kburtram**

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -132,14 +132,14 @@
...
-							"version": "0.1.5",
-							"lastUpdated": "2/27/2019",
+							"version": "0.2.0",
+							"lastUpdated": "10/2/2019",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/ServerReports/server-report-0.1.5.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json]: Were you planning on updating the whoIsActive extension as well? That was changed to not use sqlops.

File: extensionsGallery.json [hunk 2]
```diff
@@ -1740,14 +1740,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.0.1",
-							"lastUpdated": "5/31/2019",
+							"version": "0.0.2",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json]: Why are we updating all these other ones? For example admin-tool-ext was only updated to use the new iKey but that's not something we need for the October release. Same for some of the other ones.

File: extensionsGallery-insider.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
 					"versions": [
 						{
 							"version": "0.43.0",
-							"lastUpdated": "9/12/2019",
+							"lastUpdated": "10/2/2019",
```
[the remark this answers is not in the document]
Comment (kburtram) [reply extensionsGallery-insider.json]: I just make the insiders feed match the stable feed when I publish.  The dates are used anywhere. so it doesn't matter.

File: extensionsGallery.json (hunk 2 above)
Comment (kburtram) [reply extensionsGallery.json]: yeah, I was trying to make the insiders match the stable, but realized these changes may not be ready.  I move out the ones that don't have the functional changes we want to publish.

**Comment by kburtram** [comment]
@Charles-Gagnon do you have an updated whoseactive VSIX to publish?  I don't see one on the blob storage.

**Pull request merged**